blogPosts/psrules-june-2022.md

```diff
@@ -0,0 +1,144 @@
+
+| Level | Title | Description | How |
+| -- | -- | -- | -- |
+| Basic | Template validation | For IAC templates to work, they need to be valid. This could be as simple as checking the JSON structure for an ARM template and Parameter file. Or running the `bicep build` command to verify the bicep modules can correctly render into an ARM template. | - Bicep: Use `bicep build` on all `.bicep` files in the repository (find the files and do a foreach loop). <br> - ARM: Validate the ARM template and Parameter file is valid JSON. |
+| Intermediate | Compare against the Well Architected Framework (WAF) | By Validating the IAC templates pass the standards outlined in the WAF it enables organisations to analyize their templates for best practices against the 5 WAF pillars (cost, security, reliability, operational excellence and performance). These findings can help to improve the quality of your IAC templates and help detect security or reliability risks before they're implemented (`by testing on the left (in the pipeline)`). | ARM & Bicep: [PsRules For Azure](https://azure.github.io/PSRule.Rules.Azure/about/). |
+| Advanced | Validate IAC against your organisations standards | Each organisation has a different set of recommendations and standards. However it can be challenging to enforce these standards. For the most part these standards are either enforced in the IAC template or just documented. What if there was a tool that allows you to create rules to help enforce these standards (for free?!) | ARM & Bicep: [PSRules](https://microsoft.github.io/PSRule/v2/quickstart/standalone-rule/). |
+
+In this blog post we're going to focus on using **PsRules for Azure**. A blog post will be written in the future for **PsRules**, however for more information, please use the link provided below.
+
+### PsRules For Azure - (Intermediate)
+
+So we now know that PsRules for Azure can verify our IAC templates against best practices. But what is it scanning for? Well, it includes over 260 rules that it evaluates for. For a list of what is evaluated [click here](https://azure.github.io/PSRule.Rules.Azure/en/rules/module/). 
+
+Finally, let's dive into how to use this tool!
+
+#### Requirements
+
+The following platforms are supported:
+
+- Windows PowerShell 5.1 with .NET Framework 4.7.2 or greater.
+- PowerShell 7.2 or greater on MacOS, Linux, and Windows.
+
+#### Step 1 - Installation
+
+PsRules for Azure can be installed locally by running the following commands in a PowerShell terminal:
+
+To install PSRule for Azure for the current user use:
+
+    ```powershell
+    Install-Module -Name 'PsRule' -Repository PSGallery -Scope CurrentUser -Force
+    Install-Module -Name 'PSRule.Rules.Azure' -Repository PSGallery -Scope CurrentUser -Force
+    ```
+
+To enable these new modules for the current user use:
+
+    ```powershell
+    Update-Module -Name 'PSRule'
+    Update-Module -Name 'PSRule.Rules.Azure'
+    ```
```

Comment:
Markdown formatting is not correct. Either remove the language identifier or remove the indent to fix.